warble/functions.cc

```diff
@@ -24,9 +24,35 @@ std::vector<std::string> FindHashtag(Database* db, std::string hashtag) {
         warbles.push_back(id);
       }
   }
+  LOG(INFO) << "Found warble ids : " << warbles.size() << " (" << hashtag << ")";
```

Comment:
It may make sense to just return the optional ids here instead of copying the entire contents of ids into warbles.

Reply:
Yes, it just returns a list of warble ids which is then used to fetch the contents from the database later